src/installRequiredPanel.ts

```diff
@@ -0,0 +1,59 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as vscode from 'vscode';
+
+const JavascriptHrefPlaceholder : string = '[JAVASCRIPT_HREF]';
+const CssHrefPlaceholder : string = '[CSS_HREF]';
+
+export class InstallRequiredPanel {
```

Comment:
Do we need a custom UI for this? What I've typically seen in VSCode is use an information message. here's an [example from Powershell extension](https://github.com/PowerShell/vscode-powershell/blob/dba7d83e91ab2b0d1e359f8f22237450437c1f17/src/features/UpdatePowerShell.ts#L79). I especially like how it provides the user the option to update now, not update now but get reminded in future, and not to get bugged about it ever again.

Reply:
good call. probably not right to have a "don't remind me ever again" option in this case though (as the UI would just seem broken for no reason after selecting that, and the user may forget why)

Reply:
Maybe we prompt to install when the user attempts to start a neo express instance via right click or the play button. Then the options would be "install" or "cancel" with a clear user action taken to trigger the prompt again

Reply:
That's what this currently does, except that the options are "Cancel" (I don't get to choose the wording on that button) and "Show installation instructions" (do you want that to say "Install" instead?)

I also have the same behavior for the other dialogs that currently invoke neo-express, but that's the right thing to do for now and can be removed as the neo-express dependency is removed from each of those dialogs.

Reply:
Could we have Cancel, "Install" (which simply invokes `dotnet tool install Neo.Express -g`) and "More Info" (which does what showInstructions does now)?